src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportApprovedEpisodes.cs

```diff
@@ -122,7 +122,7 @@ public List<ImportResult> Import(List<ImportDecision> decisions, bool newDownloa
 
                     if (downloadClientItem != null)
                     {
-                        _eventAggregator.PublishEvent(new EpisodeImportedEvent(localEpisode, episodeFile, newDownload, downloadClientItem.DownloadClient, downloadClientItem.DownloadId, downloadClientItem.IsReadOnly));
+                        _eventAggregator.PublishEvent(new EpisodeImportedEvent(localEpisode, episodeFile, newDownload, downloadClientItem.DownloadClient, downloadClientItem.DownloadId, !downloadClientItem.CanBeDeleted));
```

Comment:
You should also change the parameter and property in EpisodeImportedEvent to match the new description. Not sure where EpisodeImportedEvent.IsReadOnly is used for. (@markus101 any idea? I'm thinking maybe it was used in the past)